Vagrantfile

```diff
@@ -28,6 +28,25 @@ Vagrant.configure("2") do |config|
     puppet.options = "--verbose"
   end
 
+  (1..2).each do |number|
+    config.vm.define "pod#{number}" do |dev|
+      # The "diaspora-pod" box is a repack of ubuntu-14.04 provisioned with diaspora-replica and
+      # deployed with capistrano. This may be used as a base box to speed up provisioning of the
+      # boxes for the test suite. But nothing must change if you disable this and fall back to
+      # "puppetlabs/ubuntu-14.04-64-puppet".
+      dev.vm.box = "diaspora-pod"
+      dev.vm.box_url = "ftp://anonymous:@188.32.218.161/usb1_1/package.box"
```

Comment:
It was very useful for my  usecase to speed up the deployment, but pointing to some extra url is not a very good thing. I think I may automate the creation of the speed-up box and make it optional.